lib/WP_Auth0_Lock_Options.php

```diff
@@ -144,7 +144,12 @@ protected function build_settings( $settings ) {
 			$options_obj['dict'] = $settings['language'];
 		}
 
-		
+		if ( $this->_is_valid( $settings, 'wpml_support' ) ) {
```

Comment:
This class is actually being deprecated in the next release. It's currently used for passwordless but that implementation is changing. You can leave this change out.

Reply:
@bjoly-nexurium - I think you got my feedback backwards. This block should be removed, the other block in `lib/WP_Auth0_Lock10_Options.php` should remain but be simplified.